src/components/Modal.js

```diff
@@ -19,6 +19,12 @@ export default class Modal {
     }
   };
 
+  setLoading(isLoading) {
+    isLoading
+      ? (this._saveButton.textContent = "Saving...")
+      : (this._saveButton.textContent = "Saved.");
```

Comment:
Please, delete `setLoading` from `Modal`

There should **not** be any code in universal functions (classes) that doesn’t suit every popup.


The image modal is a child of `Modal` but it doesn't have any submit button